api/resources_portal/models/material.py

```diff
@@ -91,3 +93,30 @@ def has_publication(self):
 
     def has_pre_print(self):
         return not (self.pre_print_doi == "" and self.pre_print_title == "")
+
+
+@receiver(post_save, sender="resources_portal.Material")
+def fix_attachment_organizations(
+    sender, instance=None, created=False, update_fields=None, **kwargs
+):
+    """If the organziation changes, update the attachment.
+    """
+    if created or not instance:
+        # Nothing to do during creation.
+        return
+
+    attachments_needing_update = instance.sequence_maps.exclude(owned_by_org__isnull=True).exclude(
```

Comment:
I can't remember why I excluded orgs attachments that had null orgs. If they're associated with a material then they should be owned by the same org as the material right? I think I should change this.
```suggestion
    attachments_needing_update = instance.sequence_maps.exclude(
```